src/server/controllers/index.ts

```diff
@@ -27,7 +27,7 @@ export function setupApiRoutes(app: express.Application): void {
   app.get('/api/users/me', authMiddleware, userController.getMe);
   app.post('/api/users/register', validate(userController.registerValidation), userController.registerUser);
   app.put('/api/users/resetpassword', validate(userController.resetPasswordValidation), userController.resetPassword);
-  app.put('/api/users/changepassword', authMiddleware, validate(userController.changePasswordValidation), userController.changePassword);
+  app.put('/api/users/changepassword', validate(userController.changePasswordValidation), userController.changePassword);
```

Comment:
I was thinking that this will be used when the user is logged in. When the user is authenticated I don't think he needs an email. Like an "Account Settings" form with:
```
oldPassword
newPassword
repeatPassword
```

But maybe we can change it in the future and use the same endpoint, if the user passes a valid token to the API (meaning he is logged in), he will be able to change the password directly by passing the old and the new password. What do you think?